upload/src/addons/SV/WarningImprovements/Setup.php

```diff
@@ -183,6 +183,37 @@ public function uninstallStep2()
         }
     }
 
+    /**
+     * @param Create|Alter $table
+     * @param string       $name
+     * @param string|null  $type
+     * @param string|null  $length
+     * @return \XF\Db\Schema\Column
+     */
+    protected function addOrChangeColumn($table, $name, $type = null, $length = null)
```

Comment:
If you add a column which exists and does not have the right schema, XF2 will rename it to __conflict and create an empty replacement.

In our case we want to migrate the schema (if the column exists) or add it (if it does not). This helper function does that.